app/models/experiment.rb

```diff
@@ -762,7 +762,7 @@ def update_module_groups(module_groups, current_user)
     dg.to_undirected.each_connected_component { |w| workflows <<  w }
 
     # Retrieve maximum allowed module group name
-    max_length = (MyModuleGroup.validators_on(:name).select { |v| v.class == ActiveModel::Validations::LengthValidator }).first.options[:maximum]
```

Comment:
Like the ingenuity of the original solution. Don't know what it does, don't know how it does it, and definitely don't know why it does it like so, but it looks pretty awesome!